docs/store/README.md

```diff
@@ -20,6 +20,7 @@ Users can use the following software solutions to store data on the Filecoin net
 
 The following applications allow you to store data on the Filecoin network with no technical setup:
 
+- [Web3.Storage](https://web3.storage) is an IPFS pinning service and Filecoin storage platform built-in-one. Upload files to Web3.Storage and access them instantly, safe in the knowledge that your data is securely backed up using the Filecoin network.
 - [Slate](slate.md) allows uploading and storing content on the Filecoin network directly from your browser. It supports one time deals, is perfect for creating image galleries, and the first 50 GB of storage is free!
```

Comment:
```
[Slate](slate.md) allows uploading and storing content on the Filecoin network directly from your browser. It supports one time deals, is perfect for creating image galleries, and the first 50 GB of storage is free!
```

Lets change this to

```
[Estuary](https://estuary.tech) allows uploading and storing content on the Filecoin network directly from your browser, command line, and API.
```

Reply:
Do you'd like to _remove_ Slate here and replace it with Estuary, correct?

```suggestion
[Estuary](https://estuary.tech) allows uploading and storing content on the Filecoin network directly from your browser, command line, and API.
```

Reply:
@jimmylee can you confirm?